docs/readmes/feg/s1ap_federated_tests.md

```diff
@@ -195,14 +192,16 @@ vagrant up magma_trfserver
 Once you have built all of the VMs, you can try to run the tests from the
 `magma_test` VM.
 
+##### Note: Only [these LTE integ](https://github.com/magma/magma/blob/master/lte/gateway/python/integ_tests/defs.mk#L288) tests can be run in federated mode
```

Comment:
I guess it would be nicer to use something along the lines of skipif (see https://docs.pytest.org/en/7.1.x/how-to/skipping.html). To just not execute the tests. 

Or even better a custom decorator like it is shown in this example: https://stackoverflow.com/questions/67200501/pytest-run-only-tests-with-specific-marker-attribute